HTML/1. intro.md

```diff
@@ -0,0 +1,38 @@
+# Intro
+
+![HTML](https://img.17qq.com/images/mhsgsnffsky.jpeg)
+
+HTML is a markup language, meaning that it is used to create webpages, so it's not like a programming language like Python
+___
+HTML is structured by tags, which make elements, here is an example of a tag - `<html>`. And here is an example of an element `<html></html>`
+
+Each element has a opening tag and a closing tag, the opening tag has a `<` to begin and a `>` to end. The closing tag has a `<` to begin and a `/>` to end. Everything else between the opening and closing tags is either content or other nested elements
```

Comment:
How about, instead of introducing `<element>...</element>` as the normal syntax, and void elements (`<element>`) as an exception, just introducing them side-by-side, with their appropriate syntaxes, more as equals?